gantry/core/axis_type.cpp.in

```diff
@@ -1,3 +1,3 @@
 #include "gantry/core/axis_type.hpp"
 
-constexpr auto gantry_type = can_ids::NodeId::${GANTRY_AXIS_TYPE};
```

Comment:
I had to hack this code gen bit to get it to work. 

I feel that the code generation should make a header file and not a cpp file. @ahiuchingau ?

Reply:
We could generate a header but it would then have to be header-safe, meaning
- a `using` declaration
- a `#define`
- an inline function-definition

If we did that, changing the define would incur a recompile of everything that includes the header. IME, codegen that sets concrete values in a cpp file that is then retrieved in a way that doesn't change based on the codegen is the right way to do that.